visivo/commands/run_phase.py

```diff
@@ -25,8 +28,9 @@ def changed(trace):
     traces = list(filter(changed, traces))
 
     Logger.instance().info(
-        f"Running project with {len(traces)} traces(s) across {threads} threads.\n"
+        f"Running project with {len(traces)} traces(s) across {threads} threads and default target {default_target}\n"
```

Comment:
Probably only worth adding this message if "default_target" is present.

Reply:
Good point, adding that and then merging 👍